frontend/packages/shared/src/components/altinnHeader/AltinnHeader.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import { render as rtlRender, screen } from '@testing-library/react';
+import { AltinnHeader, IAltinnHeader } from './AltinnHeader';
+import {
+  ServicesContextProps,
+  ServicesContextProvider,
+} from 'app-development/common/ServiceContext';
+
+describe('AltinnHeader', () => {
+  it('should render AltinnHeaderMenu', () => {
+    render();
+    expect(screen.getByTestId('altinn-header-menu')).toBeInTheDocument();
+  });
+
+  it('should render AltinnHeaderButtons', () => {
+    render();
+    expect(screen.getByTestId('altinn-header-buttons')).toBeInTheDocument();
+  });
+
+  it('should render AltinnSubMenu', () => {
+    render();
+    expect(screen.getByTestId('altinn-sub-menu')).toBeInTheDocument();
+  });
```

Comment:
`AltinnSubMenu` skal bare lastes inn hvis `showSubMenu` er `true`. For å dekke alle situasjoner, bør vi også ha en test som sjekker at den _ikke_ lastes inn hvis `showSubMenu` er `false`.

Reply:
Har lagt en ny test til /false/ situasjon